### ISSUE TITLE
How does search with uuid work?

### ISSUE THREAD

**Issue by BharatKalluri** [pschmitt/pykeepass#61]
I have tried doing 
``` python
DB.find_entry_by_uuid(DB.find_entries_by_username("username")[1].uuid)
```
but I get the folllowing error
```
ValueError: badly formed hexadecimal UUID string
```
Sorry if this is a noob question, but how can I get the uuid?

Thanks for the library @pschmitt , This is one of the most well documented and complete libs about keepass!

**Comment by Evidlo** [comment]
This was fixed in https://github.com/pschmitt/pykeepass/pull/59.  The function was also renamed to `find_entries_by_uuid`. 

I don't think it's been uploaded to Pypi yet though.

**Comment by BharatKalluri** [comment]
Thanks @Evidlo !

I hope this gets pushed to the pypi as soon as possible. I am making a keepass client for linux using gtk and this is the only feature missing.

Cheers!

**Comment by Evidlo** [comment]
In the meantime you could do this:

```
def my_find_entry_by_uuid(kp, uuid, tree=None, history=False):

    xp = './/Entry/UUID[text()="{}"]/..'.format(uuid)

    res = kp._xpath(tree=tree, xpath_str=xp)

    if history is False:
        res = res[0]

    return res

my_find_entry_by_uuid(DB, DB.find_entries_by_username("username", first=True).uuid)
```

**Comment by BharatKalluri** [comment]
Thanks for the quick fix. I will close this issue, but I hope the update gets pushed to pypi as soon as possible :relaxed:

**Comment by pschmitt** [comment]
I just released version 2.8.0 which contains the mentionned changes.
